Road/notebooks/feature_selected.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data['year'], data['month'], data['day'], data['hour'], data['minute'], data['second'] = FS.extract_dates(date=data['date'])
</pre><p>The line is too long based on PEP8 convention.</p><p>Could you please consider reducing the length of the lines by breaking it into multiple lines? like this, for example:</p><p><code>data['year'], data['month'], data['day'], \</code></p><p><code>data['hour'], data['minute'], data['second'] = FS.extract_dates(date=data['date'])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I refactor it in another way</p>